content/en/tests/auto_test_retries/_index.md

```diff
@@ -59,13 +60,30 @@ After you have set up Test Visibility, you can configure Auto Test Retries from
 The default behavior of the feature is to retry any failing test case up to 5 times.
 
 #### Known limitations
+
 [jest-image-snapshot][2] is incompatible with `jest.retryTimes` unless `customSnapshotIdentifier` is passed (see [jest-image-snapshot docs][3]) to `toMatchImageSnapshot`. Therefore, auto test retries do not work unless `customSnapshotIdentifier` is used.
 
 [1]: /tests/explorer/
 [2]: https://www.npmjs.com/package/jest-image-snapshot
 [3]: https://github.com/americanexpress/jest-image-snapshot?tab=readme-ov-file#jestretrytimes
 {{% /tab %}}
 
+{{% tab "Ruby" %}}
+After you have set up Test Visibility, you can configure Auto Test Retries from the [Test Service Settings page][1].
```

Comment:
The `Test Service Settings` link points to Test Visibility Explorer docs. It'd be better, I think, to point it directly to the settings page in the Datadog web-app. The link is incorrect for the other languages too.

Reply:
The problem with Datadog web app is that we have different web apps for different regions, should we link to datadoghq.com or datadoghq.eu? Maybe it is better to link to https://docs.datadoghq.com/tests/setup/ ?

Reply:
Then I propose to leave like it is for now

Reply:
Seeing as this is not merged yet, I got my answer from the docs team:

> you can use http://app.datadoghq.com/ and it should switch for the user (as long as they have the right Datadog site selected from the dropdown) 

So we can use the `https://app.datadoghq.com/ci/settings/test-service` link here.
Feel free to merge as is though, I will update it in a separate PR if needed